docs/learn/avalanche/avalanche-platform.md

```diff
@@ -2,33 +2,35 @@
 description: Avalanche features 3 built-in blockchains which includes Exchange Chain (X-Chain), Platform Chain (P-Chain), and Contract Chain (C-Chain).  More info here.
 ---
 
-# Avalanche Platform
+# The Primary Network
 
 Avalanche is a heterogeneous network of blockchains. As opposed to homogeneous networks, where
 all applications reside in the same chain, heterogeneous networks allow separate chains to be
 created for different applications.
 
-The Primary Network is a special [Subnet](../../subnets/README.md) that contains all validators
-(including validators of any custom Subnets). A node can become a validator for the Primary
-Network by staking at least 2,000 AVAX.
+The Primary Network is essentially a [Subnet](subnets-overview.md) that runs three blockchains:
+
```

Comment:
can we cut "essentially" from this sentence and say exactly what it is. I think the previous wording of a "special" subnet is more clear.